ui-tests/src/test/java/org/openmrs/reference/RecordMergeIssueTest.java

```diff
@@ -41,9 +42,10 @@ public void setUp() throws Exception {
         patient1 = new TestPatient();
     }
 
-    @Ignore
+    @Test
+    @Category(BuildTests.class)
     public void recordMergeIssueTest() throws Exception {
-        homePage.goToRegisterPatientApp();
```

Comment:
is this change necessary,

Reply:
> is this change necessary,

Explanation about the change is already [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/461#discussion_r654835119)